themes/mcf/components/HeroComponents/MultilateralClimateFundsLogos.tsx

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+import React from "react";
+import { ExternalLink } from "@components/ExternalLink";
+
+interface FundData {
+  name: string;
+  url: string;
+  imgSrc: string;
+}
+
+export const MultilateralClimateFundLogos = () => {
+  const fundInformation: FundData[] = [
```

Comment:
Define collection outside of the component else it will redefine it on every re-render